konig-maven-common/src/main/java/io/konig/maven/TableShapeGeneratorConfig.java

```diff
@@ -0,0 +1,44 @@
+package io.konig.maven;
+
+/*
+ * #%L
+ * Konig Maven Common
+ * %%
+ * Copyright (C) 2015 - 2018 Gregory McFall
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+
+
+import org.apache.maven.model.FileSet;
+
+public class TableShapeGeneratorConfig {
```

Comment:
Eliminate the classes `TableShapeGeneratorConfig` and `ViewShapeGeneratorConfig`.  We should use instead the class `TabularShapeGeneratorConfig` which I added to your branch.  This new class provides common configuration for both the TableShapeGenerator and the ViewShapeGenerator.